docs/types.md

```diff
@@ -44,11 +44,11 @@ zsh: command not found: s
 
 ## Global
 
-Regular abbreviations expand everywhere.
+Global abbreviations expand everywhere.
 
 To scope a zsh-abbr command to the user, pass the `--global` flag or its shorthand `-g`.
 
-In the following examples, global abbreviations expand at the start of the line but not in other positions:
+In the following examples, global abbreviations can expand at any positions:
```

Comment:
What do you think about something like

```suggestion
In the following examples, global abbreviations expand at the start of the line and also in other positions:
```

to most directly contrast with the `regular` documentation?

If you don't like that, let's at least do

```suggestion
In the following examples, global abbreviations expand in any position:
```

Reply:
Thank you for your feedback! Yes, it's okay with me. I'll merge the suggestion.